test/components/CodeCell.spec.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { spy } from 'sinon';
+import { shallow } from 'enzyme';
+
+describe('CodeCell', () => {
+    it('notifyRender should call props.onRender', () => {
+        expect(true).toBe(true); // just to get a test to pass
+    });
+});
```

Comment:
It's better to leave a blank tailing line at each file, maybe we need a `.editorconfig` file

Reply:
You're right. I'm used to running eslint autoFixOnSave in my projects which handles things like trailing spaces and trailing line at the end of files.

 Maybe we can look into adding this at some point?

Reply:
Done.